src/Song.cpp

```diff
@@ -457,18 +462,22 @@ bool Song::LoadFromSongDir(RString sDir, bool load_autosave, ProfileSlot from_pr
 	return true;	// do load this song
 }
 
+

```

Comment:
I think we can remove these extra new lines (and the one at the end of the file).